app/code/Bulbulatory/Recommendations/Controller/Customer/Recommendation.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Bulbulatory\Recommendations\Controller\Customer;
+
+use Magento\Framework\App\Action\Action;
+
+/**
+ * Class Recommendation
+ * @package Bulbulatory\Recommendations\Controller\Customer
+ */
+class Recommendation extends Action
+{
+    public function execute()
+    {
+        $this->_view->loadLayout();

```

Comment:
It's possible to display this page (and also send recommendation) if the functionality is disabled in the configuration. It should be checked if the module is enabled or not and display the page only when it is. Otherwise either display 404 or redirect to homepage should happen.

Reply:
@TM18

> It's possible to display this page (and also send recommendation) if the functionality is disabled in the configuration. It should be checked if the module is enabled or not and display the page only when it is. Otherwise either display 404 or redirect to homepage should happen.

Fixed! I created for check access new **Bulbulatory\Recommendations\Helper\Acl** class